@here/harp-mapview/lib/geometry/TileGeometryManager.ts

```diff
@@ -4,15 +4,39 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { GeometryKind, GeometryKindSet } from "@here/harp-datasource-protocol";
 import { MapView } from "../MapView";
 import { Tile } from "../Tile";
 import { SimpleTileGeometryLoader, TileGeometryLoader } from "./TileGeometryLoader";
 
 /**
  * Manages the content (the geometries) of a tile. Derived classes allow different strategies that
- * control the sequence in which the geometries of the tile are being loaded.
+ * control the sequence in which the geometries if the tile are being loaded.
```

Comment:
geometries of the tile

Reply:
I was just checking if somebody reads this... 8^)